internal/website/content/howto/runtimervar/runtimevar.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Fetching the latest value of a variable"
+date: 2019-06-18T15:46:26-07:00
+---
+
+Working with runtime variables using the Go CDK takes two steps:
+
+1. Open the variable with the `runtimevar` provider of your choice.
+2. As many times as needed, use the `Latest` or `Watch` methods to fetch the
+   value of the variable.
+
+## Constructors versus URL openers
+
+The easiest way to open a variable is using [`runtimevar.OpenVariable`][] and a
+URL pointing to the variable. See the [documentation on URLs][] for more
+details. If you need fine-grained control over the connection settings, you can
+call the constructor function in the driver package directly (like
+`awsparamstore.OpenVariable`). This guide will show how to use both forms for
+each storage provider.
+
+[documentation on URLs]: {{< ref "/concepts/urls.md" >}}
+
+[`runtimevar.OpenVariable`]:
+https://godoc.org/gocloud.dev/runtimevar#OpenVariable
+
+## GCP Runtime Configurator {#rc-url}
```

Comment:
Optional: Consider using GCP/AWS/Azure for these fragments, since these acronyms aren't well-accepted.

```suggestion
## GCP Runtime Configurator {#gcp-url}
```